cluster/addons/fluentd-gcp/event-exporter.yaml

```diff
@@ -29,32 +29,32 @@ subjects:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: event-exporter-v0.2.5
+  name: event-exporter-v0.3.0
   namespace: kube-system
   labels:
     k8s-app: event-exporter
-    version: v0.2.5
+    version: v0.3.0
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   replicas: 1
   selector:
     matchLabels:
       k8s-app: event-exporter
-      version: v0.2.5
+      version: v0.3.0
   template:
     metadata:
       labels:
         k8s-app: event-exporter
-        version: v0.2.5
+        version: v0.3.0
     spec:
       serviceAccountName: event-exporter-sa
       containers:
       - name: event-exporter
-        image: k8s.gcr.io/event-exporter:v0.2.5
+        image: k8s.gcr.io/event-exporter:v0.3.0
         command:
         - /event-exporter
-        - -sink-opts=-stackdriver-resource-model={{ exporter_sd_resource_model }}
+        - -sink-opts=-stackdriver-resource-model={{ exporter_sd_resource_model }} -endpoint={{ exporter_sd_endpoint }}
```

Comment:
Is it not going to be translated to:
/event-exporter -sink-opts=-stackdriver-resource-model={{ exporter_sd_resource_model }} -endpoint={{ exporter_sd_endpoint }}

I think we need to change it to:
/event-exporter -sink-opts="-stackdriver-resource-model={{ exporter_sd_resource_model }} -endpoint={{ exporter_sd_endpoint }}"

Reply:
I did some experiments and the above command is translated to:
`./event-exporter "-sink-opts=-stackdriver-resource-model={{ exporter_sd_resource_model }} -endpoint={{ exporter_sd_endpoint }}"`
Which is equivalent to
`./event-exporter -sink-opts="-stackdriver-resource-model={{ exporter_sd_resource_model }} -endpoint={{ exporter_sd_endpoint }}"`